MoneyDataType/MoneyDataType.csproj

```diff
@@ -1,7 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>Money</RootNamespace>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Authors>Allan Nielsen</Authors>
+    <Company>Our Presence</Company>
+    <Product>MoneyDataType</Product>
+    <Description>New data type representing monetary values.</Description>
+    <Copyright>Copyright © 2019 Our Presence</Copyright>
```

Comment:
Yeah, well, I think I wrote a fair share of that code ;)

Reply:
Sorry, i malice intended, happy to put your details against it, maybe it's easier if you fill in your details after?

Reply:
Sure. Let's attribute the copyright to the .NET Foundation like the rest of Orchard however and remove the company tag. And well, while you're at it, just add my name. :D

Reply:
Updated, but didn't realise I could add .Net Foundation as the company.  This hasn't been approved for inclusion into the foundation has it?

Reply:
This project is officially part of Orchard, which is a .NET Foundation project.